core/src/test/resources/simplelogger.properties

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is this necessary?

Reply:
right removed it, it was useful at the beginning for understanding the code.